ipn/ipnlocal/c2n.go

```diff
@@ -337,8 +337,16 @@ func handleC2NPostureIdentityGet(b *LocalBackend, w http.ResponseWriter, r *http
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		}
-
 		res.SerialNumbers = sns
+
+		// TODO: one we are confident with this, remove this GET parameter
```

Comment:
Typo: once

It'd be good to document how the future reader can tell when we are confident enough (e.g., it survives some number of releases, or whatever).

As a start, can we include a date and/or an issue number here, too? E.g., 
```go
// TODO(tailscale/corp#21371, 2024-07-10): Once this has landed in a stable release
// and looks good in client metrics, remove this parameter and always report MAC
// addresses.
```

or whatever is true.